imap_processing/swapi/l1/swapi_l1.py

```diff
@@ -105,7 +105,7 @@ def decompress_count(
     # Decompress counts based on compression indicators
     # If 0, value is already decompressed. If 1, value is compressed.
     # If 1 and count is 0xFFFF, value is overflow.
-    new_count = copy.deepcopy(count_data).astype(np.int32)
+    new_count: np.ndarray = copy.deepcopy(count_data).astype(np.int32)
```

Comment:
I messed up when I added this earlier, could you change this while you're at it? We don't need to deepcopy, the default array constructor is to make a copy of the data. https://numpy.org/doc/stable/reference/generated/numpy.array.html

```suggestion
    new_count = np.array(count_data, dtype=np.int32)
```